arma3/@task_force_radio/addons/task_force_radio/functions/fn_requestRadios.sqf

```diff
@@ -60,9 +60,17 @@ if ((time - TF_last_request_time > 3) or {_this}) then {
 		_copyIndex = 0;
 		if ((typename _response) == "ARRAY") then {
 			private ["_radioCount","_settingsCount", "_startIndex"];
+
+			{
+				if (_x == "ItemRadio" || _x in _radiosToRequest) then {
+					TFAR_currentUnit unassignItem _x;					
+					TFAR_currentUnit removeItem _x;					
+				};
+			} forEach items TFAR_currentUnit;
```

Comment:
Just moved code from radioToRequestCount.

Old sequence:
1) remove items
2) call to server
3) add items

if player kicked between 2 and 3 he will receive no radio at all.

I've changed sequence to:
1) call to server
2) remove items
3) add items